pages/home/passkeys-guides/safe-sdk.mdx

```diff
@@ -1,7 +1,11 @@
-import { Steps, Tabs } from 'nextra/components'
+import { Callout, Steps, Tabs } from 'nextra/components'
 
 # Passkeys with the Safe\{Core\} SDK
 
+<Callout type='warning' emoji='⚠️'>
+  Passkeys support is still under audit and should not be used in production environments. Please consider this tutorial and all the code examples in it a developer preview. 
```

Comment:
Should we mention that is only available for alpha when using Sepolia?

```suggestion
  Passkeys support is still under audit and should not be used in production environments. Please consider this tutorial and all the code examples in it a developer preview.
  Compatibility with passkeys only available when using the kits alpha in Sepolia.
```

Reply:
Done! It was specified in the prerequisites, but this way becomes much more visible.